ui/.node-version

```diff
@@ -0,0 +1 @@
+16.15.0
```

Comment:
I don't think `.node-version` is a common dotfile for node version selection.

Probably `.nvmrc` is a better dotfile which can be read by [`nvm`](https://github.com/nvm-sh/nvm#nvmrc).

And we may use `lts/gallium` as the content of `.nvmrc`.

Reply:
Replaced it with `package.json`